js/ccf-app/src/endpoints.ts

```diff
@@ -141,13 +141,32 @@ export interface MemberCertAuthnIdentity extends UserMemberAuthnIdentityCommon {
   policy: "member_cert";
 }
 
+interface UserMemberCOSEAuthIdentityCommon {
```

Comment:
Not sure the UserMember prefix adds much here?

Reply:
I followed the same pattern as `UserMemberAuthnIdentityCommon`. Happy to change it though.